ivy_tests/test_ivy/test_frontends/test_jax/test_jax_numpy_searching_sorting.py

```diff
@@ -278,3 +278,29 @@ def test_numpy_sort(
         a=x[0],
         axis=axis,
     )
+
+
+# flatnonzero
+@handle_frontend_test(
+    fn_tree="jax.numpy.flatnonzero",
+    dtype_and_x=helpers.dtype_and_values(
+        available_dtypes=helpers.get_dtypes("float"),
```

Comment:
We should test for all numeric dtypes I am sure that this function should support it